canonical-web-and-design/ubuntu.com
# Add header buttons

## Done

- Add header buttons.
- Make header responsive.

## QA

- Visit /advantage?test_backend=true.
- Clicking the buttons should take you to the appropriate place.
- Resize your screen down to the small size and the buttons should drop below the title and become stacked.


## Issue / Card

Fixes: canonical-web-and-design/commercial-squad#88.
Fixes: canonical-web-and-design/commercial-squad#89.

## Screenshots

<img width="1110" alt="Screen Shot 2021-08-04 at 1 36 12 pm" src="https://user-images.githubusercontent.com/361637/128118370-a25b459f-d6a6-4208-957e-39f7f3e7e5e2.png">
<img width="342" alt="Screen Shot 2021-08-04 at 1 36 24 pm" src="https://user-images.githubusercontent.com/361637/128118375-6fa0decc-a0d1-4c8f-9f7e-588281c4f081.png">
